api/_mixins/ParentNode__DocumentFragment.json

```
@@ -431,7 +431,7 @@
               "version_added": "14"
             },
             "safari_ios": {
-              "version_added": "14"
+              "version_added": "14.5"
```

```suggestion
              "version_added": "14"
```

As above, but tested with https://mdn-bcd-collector.appspot.com/tests/api/DocumentFragment/replaceChildren